pkgs/by-name/ag/ags/package.nix

```diff
@@ -63,11 +63,18 @@ buildNpmPackage rec {
     chmod u+x ./post_install.sh && patchShebangs ./post_install.sh
   '';
 
+  enableParallelBuilding = true;
```

Comment:
Apparently `meson` sets this to true already.

https://ryantm.github.io/nixpkgs/stdenv/stdenv/#var-stdenv-enableParallelBuilding

Reply:
Done. Thanks!